src/reactivity/typeHandlers.ts

```diff
@@ -114,6 +119,7 @@ function defaultHandlers(options: ReactiveOptions) {
       return value;
     },
     set: (target: object, propertyKey: PropertyKey, value: any) => {
+      console.log('set', propertyKey);
```

Comment:
Do you want this console.log here? Seems unlikely 😉

Reply:
whoops 😬  good catch, thanks!